uber/templates/signups/shirt_item.html

```diff
@@ -1,11 +1,18 @@
 {% import 'macros.html' as macros %}
 
 <li>
-    {{ macros.checklist_image(attendee.shirt_size_marked) }}
+    {{ macros.checklist_image(attendee.shirt_info_marked) }}
     {% if not attendee.placeholder %}
         <a href="shirt_size">Let us know</a>
     {% else %}
         Let us know
     {% endif %}
-    your shirt size so that we can set aside a shirt in your size at our merch booth.
+    your shirt size
+    {% if attendee.gets_staff_shirt %}
+        and your preference of staff or event shirt
```

Comment:
This text seems like it might be a little unclear. I'm not sure what would be better, maybe "and whether you want one of your staff shirts to be an event shirt instead" or... something.